apps/taquito-test-dapp/README.md

```diff
@@ -4,13 +4,18 @@
 
 A minimal end-to-end testing setup for developing Tezos DApps with Taquito and Beacon to manage signing and wallet operations.
 ## Getting Started
+#### Initial setup:
 1. Clone the Taquito repository: `git clone git@github.com:ecadlabs/taquito.git`
-2. Change your current working directory to the newly cloned repository directory, then change to `apps/taquito-test-dapp`.
-3. Install all dependencies:
-    `npm clean-install`
-4. Start development server:
-    `npm run dev`
-5. Open http://localhost:3030 in your browser to see a sample application.
-6. Start production server:
-    `npm run build && npm run preview`
-7. Open http://localhost:4173 in your browser to see a preview application.
+1. Change your current working directory to the newly cloned repository.
+1. Install dependencies: `npm i`
```

Comment:
We don't use `install`, but rather `clean-install` so it respects package.json and we get deterministic builds. Also, prefer using the full name of the command `clean-install` for documentation purposes, programmers that need `ci` will find it :wink: